examples/access-control-migration/src/services/casbin.authorizer.ts

```diff
@@ -0,0 +1,72 @@
+import { Provider, inject } from '@loopback/core';
+import {
+  Authorizer,
+  AuthorizationContext,
+  AuthorizationMetadata,
+  AuthorizationRequest,
+  AuthorizationDecision,
+} from '@loopback/authorization';
+import * as casbin from 'casbin';
+
+// Class level authorizer
+export class CasbinAuthorizationProvider implements Provider<Authorizer> {
+  constructor(@inject('casbin.enforcer.factory') private enforcerFactory: (name: string) => casbin.Enforcer) { }
+
+  /**
+   * @returns authenticateFn
+   */
+  value(): Authorizer {
+    return this.authorize.bind(this);
+  }
+
+  async authorize(
+    authorizationCtx: AuthorizationContext,
+    metadata: AuthorizationMetadata,
+  ): Promise<AuthorizationDecision> {
+    const subject = this.getUserName(authorizationCtx.principals[0].id);
+    const object = authorizationCtx.resourceId ?? metadata.resource ?? authorizationCtx.resource;
```

Comment:
`authorizationCtx.resource` not really used at all.

Reply:
see https://github.com/strongloop/loopback-next/pull/4571#discussion_r382813314 ^